### DISCUSSION TITLE
Pull replication with snapshot deletion on target?

### DISCUSSION THREAD

**Discussion by bsnipes** [psy0rz/zfs_autobackup#157] [General]
I have a TrueNAS backup system that I need to pull snapshots from to an Ubuntu backup server. I need to keep the Ubuntu backup server updated and would like to use PULL and have it keep the snapshots on the Ubuntu backup server updated including deleting snapshots that no longer exist on the TrueNAS server. The naming convention for my snapshots are auto-%Y%m%d.%H%M-(3m/1m/2w/1w) on the TrueNAS backup system. Is there a set of parameters that zfs_autobackup has that can PULL the snapshots and delete snapshots on the Ubuntu server that no longer exist on the server being pulled from? Any help is appreciated.

**Comment by psy0rz** [comment]
sorry i missed this one. did you figure it out? :)

**Reply by bsnipes** [reply]
No problem. I ended up using syncoid and created a script to clean up snapshots that didn't exist on the source server.

**Reply by psy0rz** [reply]
ok, are you still using zfs-autobackup for the regular backups then? :)

**Reply by bsnipes** [reply]
I haven't tried it yet. I was familiar with sanoid/syncoid so went with it. However, having said that, I am working on my test box at the moment so will give zfs-autobackup a try :-)

**Reply by psy0rz** [reply]
w00p \o/ :)
